src/Log/EchoLogger.php

```diff
@@ -18,71 +19,78 @@ class EchoLogger implements LoggerInterface
      */
     public function emergency($message, array $context = array())
     {
-        $this->logErrorMessage('EMERGENCY', $message, $context);
+        $this->log(LogLevel::EMERGENCY, $message, $context);
     }
 
     /**
      * @inheritDoc
      */
     public function alert($message, array $context = array())
     {
-        $this->logErrorMessage('ALERT', $message, $context);
+        $this->log(LogLevel::ALERT, $message, $context);
     }
 
     /**
      * @inheritDoc
      */
     public function critical($message, array $context = array())
     {
-        $this->logErrorMessage('CRITICAL', $message, $context);
+        $this->log(LogLevel::CRITICAL, $message, $context);
     }
 
     /**
      * @inheritDoc
      */
     public function error($message, array $context = array())
     {
-        $this->logErrorMessage('ERROR', $message, $context);
+        $this->log(LogLevel::ERROR, $message, $context);
     }
 
     /**
      * @inheritDoc
      */
     public function warning($message, array $context = array())
     {
-        $this->logErrorMessage('WARNING', $message, $context);
+        $this->log(LogLevel::WARNING, $message, $context);
     }
 
     /**
      * @inheritDoc
      */
     public function notice($message, array $context = array())
     {
-        $this->logErrorMessage('NOTICE', $message, $context);
+        $this->log(LogLevel::NOTICE, $message, $context);
     }
 
     /**
      * @inheritDoc
      */
     public function info($message, array $context = array())
     {
-        $this->printMessage('INFO', $message, $context);
+        $this->log(LogLevel::INFO, $message, $context);
     }
 
     /**
      * @inheritDoc
      */
     public function debug($message, array $context = array())
     {
-        $this->printMessage('DEBUG', $message, $context);
+        $this->log(LogLevel::DEBUG, $message, $context);
     }
 
     /**
      * @inheritDoc
      */
     public function log($level, $message, array $context = array())
     {
-        $this->printMessage('LOG', $message, $context);
+        switch ($level) {
+            case LogLevel::INFO:
+            case LogLevel::DEBUG:
+                $this->printMessage($level, $message, $context);
+                break;
+            default:
+                $this->logErrorMessage($level, $message, $context);
```

Comment:
In case if error log we wont the the result on stdout. 
My Suggestion is: Log everithing on stdout and additional evenerthing >=WARING into Error Log